recipes/poppler-data/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class PopplerDataConan(ConanFile):
+    name = "poppler-data"
+    description = "encoding files for use with poppler, enable CJK and Cyrrilic"
+    homepage = "https://poppler.freedesktop.org/"
+    topics = "conan", "poppler", "pdf", "rendering"
+    license = "GPL-2.0-or-later", "adobe", "MIT"
```

Comment:
adobe is BSD-3-Clause, but has Adobe company listed. Interesting.

Reply:
I just checked the contents, and `COPYING.adobe` is actually BSD-3-Clause.

Reply:
Do you agree on: `"BSD-3-Clause", "GPL-2.0-or-later", "MIT"`?

Reply:
@madebr Yes!

Reply:
Whitelisted in hooks https://github.com/conan-io/hooks/pull/237